mkosi/__init__.py

```diff
@@ -164,7 +164,6 @@ def install_build_packages(context: Context) -> None:
     if not context.config.build_scripts or not context.config.build_packages:
         return
 
-    # TODO: move to parenthesised context managers once on 3.10
     pd = str(context.config.distribution).capitalize()
     with complete_step(f"Installing build packages for {pd}"), mount_build_overlay(context):
```

Comment:
The idea here was this, if I remember correctly
```suggestion
    with (
        complete_step(f"Installing build packages for {str(context.config.distribution).capitalize()}"), 
        mount_build_overlay(context)
    ):
```
because including it on the line without being able to break it ran afoul the line length limit.